source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,214 @@
+
+#### Single Frame
+
+We can configure settings for an individual frame directly to the camera.
+```csharp
+camera.UpdateSettings(s =>
+{
+    s.Iris = 10;
+    s.ExposureTime = Duration.FromMicroseconds(10000);
+    s.Brightness = 1;
+    s.Gain = 1;
+    s.Bidirectional = false;
+    s.Filters.Contrast.Enabled = true;
+    s.Filters.Contrast.Threshold = 5;
+    s.Filters.Gaussian.Enabled = true;
+    s.Filters.Gaussian.Sigma = 1.5;
+    s.Filters.Outlier.Enabled = true;
+    s.Filters.Outlier.Threshold = 5;
+    s.Filters.Reflection.Enabled = true;
+    s.Filters.Saturated.Enabled = true;
+    s.BlueBalance = 1.081;
+    s.RedBalance = 1.709;
+});
+```
+
+#### HDR Frame
+
+We may also set a list of settings to be used in an [HDR capture](#capture-hdr).
+```csharp
+var irisList = new List<ulong>() { 20, 25, 30 };
+var settingsList = new List<Zivid.NET.Settings>();
+foreach (var iris in irisList)
+{
+    Console.WriteLine("Add settings for frame with iris = " + iris);
+    var settings = new Zivid.NET.Settings();
+    settings.Iris = iris;
+    settingsList.Add(settings);
+}
+```
+
+### From File
+
+Zivid Studio can store the current settings to .yml files. These can be read and applied in the API. You may find it easier to modify the settings in these (human-readable) yaml-files in your preferred editor.
+```csharp
+camera.SetSettings(new Zivid.NET.Settings("frame_01.yml"));
+```
+
+### 2D Settings
+
+It is possible to only capture a 2D image. This is faster than a 3D capture, and can be used . 2D settings are configured as follows ([go to source][settings2d-url]).
+```csharp
+var settings = new Zivid.NET.Settings2D()
+{
+    Iris = 22,
+    ExposureTime = Duration.FromMicroseconds(10000),
+    Gain = 1.0
+};
+```
+
+## Capture
+
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```csharp
+var frame = camera.Capture();
+```
+
+### Capture HDR
+
+As was revealed in the [Capture Assistant](#capture-assistant) section, a capture may consist of multiple frames. In order to capture multiple frames, and combine them, we can do as follows ([go to source][captureHDR-url])
+```csharp
+var hdrFrame{ Zivid.NET.HDR.Capture(camera, settingsList) };
+```
+It is possible to [manually create](#hdr-frame) the `settingsList`, if not set via [Capture Assistant](#capture-assistant).
+
+### Capture 2D
+
+If we only want to capture a 2D image, which is faster than 3D, we can do so via the 2D API ([go to source][capture2d-url]).
+```csharp
+var frame = camera.Capture2D(settings);
+```
+
+## Save
+
+We can now save our results ([go to source][save-url]).
+```csharp
+var resultFile = "result.zdf";
+frame.Save(resultFile);
+```
+The API detects which format to use. See [Point Cloud][kb-point_cloud-url] for a list of supported formats.
+
```

Comment:
Add section for Save 2D, like in: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598